### PR TITLE
Fix path parsing problems on Windows

### PR DESCRIPTION
**Pull request by antiagainst**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Shaderc
 
-[![Build Status](https://travis-ci.org/google/shaderc.svg)](https://travis-ci.org/google/shaderc)
+[![Build Status](https://travis-ci.org/google/shaderc.svg)](https://travis-ci.org/google/shaderc) [![Build status](https://ci.appveyor.com/api/projects/status/xcxci2mekggib1an?svg=true)](https://ci.appveyor.com/project/antiagainst/shaderc)
```
Comment (dmircevski) [inline README.md]: They should probably have different text, to suggest the difference between the two.  Maybe "Linux build status" and "Windows build status"?

File: glslc/test/glslc_test_framework.py [hunk 2]
```diff
@@ -290,10 +290,11 @@ def runTest(self):
             test_status = TestStatus(
                 process.returncode, output[0], output[1],
                 self.directory, self.file_shaders)
-            for test_method in get_all_test_methods(self.test.__class__):
-                success, message = getattr(self.test, test_method)(test_status)
-                if not success:
-                    break
+            run_results = [getattr(self.test, m)(test_status)
```
Comment (dmircevski) [inline glslc/test/glslc_test_framework.py]: Why did "test_method" become "m"?  That's not exactly an enhancement to readability...

File: libshaderc_util/src/io_test.cc [hunk 3]
```diff
@@ -53,6 +53,18 @@ TEST(IsAbsolutePathTest, Linux) {
   EXPECT_TRUE(IsAbsolutePath("/☢/g o/ogle"));
 }
 
+TEST(IsAbsolutePathTest, Windows) {
+  EXPECT_TRUE(IsAbsolutePath("\\\\Server1000\\superuser\\file"));
```
Comment (dmircevski) [inline libshaderc_util/src/io_test.cc]: Consider using [raw-string literals](http://en.cppreference.com/w/cpp/language/string_literal) here, for easier readability.

File: libshaderc_util/src/message_test.cc [hunk 4]
```diff
@@ -181,4 +181,33 @@ TEST(ParseGlslangOutputTest, FileName) {
...
 }
 
+TEST(ParseGlslangOutputTest, WindowsPath) {
+  string_piece source_name;
+  string_piece line_number;
+  string_piece rest;
+
+  EXPECT_EQ(
+      MessageType::Error,
+      ParseGlslangOutput("ERROR: C:\\path\\to\\shader.glsl:5: something wrong",
```
Comment (dmircevski) [inline libshaderc_util/src/message_test.cc]: Consider using a raw-string literal.

File: README.md (hunk 1 above)
Comment (antiagainst) [inline README.md]: This is addressed in #20.

File: glslc/test/glslc_test_framework.py (hunk 2 above)
Comment (antiagainst) [inline glslc/test/glslc_test_framework.py]: Going back to `test_method`.

File: libshaderc_util/src/io_test.cc (hunk 3 above)
Comment (antiagainst) [inline libshaderc_util/src/io_test.cc]: Done

File: libshaderc_util/src/message_test.cc (hunk 4 above)
Comment (antiagainst) [inline libshaderc_util/src/message_test.cc]: Done

**Comment by dmircevski** [comment]
+2

**Pull request merged**